lib/elastic_apm/transport/connection.rb

```diff
@@ -41,6 +41,16 @@ def initialize(config, metadata)
           headers['Authorization'] = "Bearer #{token}"
         end
 
+        # Set SSL CA Cert
+        if config.use_ssl? && config.ssl_ca_cert
+          ssl_ctx = OpenSSL::SSL::SSLContext.new
+          ssl_ctx.ca_file = config.ssl_ca_cert
+
+          # include SSL context in default options
+          client_options = HTTP.default_options.merge(ssl_context: ssl_ctx)
+          HTTP.default_options = client_options
```

Comment:
Won't this leak and lead to weird bugs if the user's app also uses `HTTP.rb`? Can we pass it when we initialize the `@client` below instead?

Reply:
That's a good callout. I went this route because I was having a hard time finding how to pass the  `ssl_context` to the client with chained methods. I'll try again.

Reply:
Thank you!